noxfile.py

```diff
@@ -0,0 +1,31 @@
+import nox
+
+
+@nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10"])
+def compile(session):
+    session.install("pip-tools")
+
+    # Needed by Python 3.5 and 3.6 in nox docker image
+    session.env["LC_ALL"] = "C.UTF-8"
+    session.env["LANG"] = "C.UTF-8"
+
+    session.run(
+        "pip-compile",
+        "--generate-hashes",
+        "requirements.in",
+        "--allow-unsafe",
+        "--output-file",
+        f"docker/build_scripts/requirements{session.python}.txt",
+    )
+
+@nox.session(python="3.7")
```

Comment:
```suggestion
@nox.session(python="3.9")
```
This has been updated between dropping Python 3.5 and adding Python 3.10